.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,14 @@
 <!--
+IMPORTANT NOTICE FOR NEW CONTRIBUTORS: This is the repository of the Python library 
+"python-telegram-bot". We do not respond to questions in other programming languages 
+or libraries such as "telepot" or "pyTelegramBotAPI".
+
 Thanks for reporting issues of python-telegram-bot!
 
 Use this template to notify us if you found a bug, or if you want to request a new feature.
 If you're looking for help with programming your bot using our library, feel free to ask your
 questions in out telegram group at: https://t.me/pythontelegrambotgroup
+We also get a lot of issues that could be answered by a quick question in the group, so please join us on Telegram.
```

Comment:
This is practically the same as the line above,

Reply:
Sure, but people obviously don't follow the advise. So I wanted to emphasize it even more